docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -16,22 +16,19 @@ When an issue comes in,
 
 # Assigning reviewers for PRs
 
-* A PR should have,
-  * **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)
-  * **all reviewers should be selected as `assignees`**
-  
-  > Duplicating `reviewers` as `assignees` is needed because GitHub filter syntax cannot filter by reviewers yet.
+* A PR should have **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)<br>
+  The `assignees` field can be left empty.
   
 * PRs created by core members:
-  * If you can, assign at least one `reviewer`. If you don't, team lead will assign reviewers for the PR.
+  * Author may request reviews from at least one core member. Else, team lead will assign reviewers for the PR.
```

Comment:
Inconsistent phrasing:
- `may` implies `>= 0`
- `at least` implies `>= 1`

How about changing `at least one core member` to `fellow core members`?